cupsfilters/ipp.c

```diff
@@ -262,10 +262,10 @@ get_printer_attributes5(http_t *http_printer,
   get_printer_attributes_log[0] = '\0';
 
   /* Convert DNS-SD-service-name-based URIs to host-name-based URIs */
-  if(!isFax)
+  if(is_fax == -1)
```

Comment:
Is it not easier to have this variable set to 0 for requesting the printing URI and 1 for requesting the fax URI (instead of -1 and 0)?